src/libraries/System.Private.Uri/tests/FunctionalTests/IriTest.cs

```diff
@@ -113,7 +107,7 @@ public void Iri_804110_TryCreateUri_ShouldNotThrowIndexOutOfRange()
                 resultUri.ToString());
 
             string u4 = "http://co.ntsosocon.com/abcd/" + "%A2%F3%BD%CB%FC%FB%D5%E9%F3%B7%AA%B5";
-            Uri uri4 = new Uri(u4);
+            _ = new Uri(u4);
```

Comment:
I still think such discards are unnecessary verbosity in ctor cases like this.

Reply:
Changed